### PR TITLE
LPS-157104 Add Filters to Aggregation Field in EditObjectField Screen

### PR DESCRIPTION
**Pull request by danvb112**
Related Issues: 

[LPS-157104 - Add Builder Screen on Edit Object Field Page](https://issues.liferay.com/browse/LPS-157104)
[LPS-157209 - Adapt Filter Modal to Support Numeric values](https://issues.liferay.com/browse/LPS-157209)
[LPS-157335 - Create functions to add and delete Filter columns on ObjectFieldSetting](https://issues.liferay.com/browse/LPS-157335)

### CODE REVIEW

**Comment by danvb112** [comment]
ci:test:sf

**Comment by danvb112** [comment]
ci:test:relevant

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/EditObjectField.tsx [hunk 1]
```diff
@@ -137,6 +157,156 @@ export default function EditObjectField({
...
+					{
+						headers: HEADERS,
+					}
+				);
+
+				const {
+					items: objectFields,
+				}: {
+					items: ObjectField[];
+				} = (await response.json()) as any;
```
Comment (alinedoleron) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/EditObjectField.tsx:301]: Hi @danvb112. Avoid using the any type. Please, check if the following change would solve it:
```suggestion
				const {
					items: objectFields,
				} = (await response.json()) as {
					items: ObjectField[];
				} ;
```

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/index.tsx [hunk 2]
```diff
@@ -106,7 +106,7 @@ const CustomView: React.FC<React.HTMLAttributes<HTMLElement>> = () => {
 			const {
 				items: objectFields,
 			}: {
-				items: TObjectField[];
+				items: ObjectField[];
 			} = (await objectFieldsResponse.json()) as any;
```
Comment (alinedoleron) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/index.tsx:110]: The same here. Avoid using any type. Sorry for that rsrs Milton would ask for this :sweat_smile:

Comment (alinedoleron) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/index.tsx:110]: The same here. Avoid using any type. Sorry for that rsrs Milton would ask for this :sweat_smile:

**Review by alinedoleron — CHANGES_REQUESTED** [review]
Good job!
There are just two notes related to type check.

**Comment by danvb112** [comment]
ci:test:sf

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/index.tsx (hunk 2 above)
Comment (danvb112) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/index.tsx]: Thanks for review @alinedoleron, I added a little more code in this last commit along with the changes you requested :laughing:

**Comment by danvb112** [comment]
ci:test:sf

**Comment by danvb112** [comment]
ci:test:sf

**Comment by danvb112** [comment]
ci:test:relevant

**Comment by danvb112** [comment]
ci:forward

**Pull request closed without merging**